CODEOWNERS

```diff
@@ -7,4 +7,4 @@
 # These owners will be the default owners for everything in the repo.
 # Unless a later match takes precedence, they will be requested for
 # review when someone opens a pull request.
-* @EGI-Foundation/admins @EGI-Foundation/operations
```

Comment:
Please add back admins, it's always useful to have this, just in case.

Reply:
sorry..hope it's fine now